akka-remote/src/main/scala/akka/remote/MessageSerializer.scala

```diff
@@ -47,4 +48,34 @@ private[akka] object MessageSerializer {
     }
     builder.build
   }
+
+  def serializeForArtery(system: ExtendedActorSystem, message: AnyRef, headerBuilder: HeaderBuilder, envelope: EnvelopeBuffer): Unit = {
+    val s = SerializationExtension(system)
```

Comment:
pass in the `SerializationExtension` as parameter instead, to avoid the extension lookup each time